### PR TITLE
Onboarding: Honyswap liquidty stable in BN

### PR DESCRIPTION
**Pull request by rperez89**

### CODE REVIEW

**Comment by rperez89** [comment]
This was already deployed to staging

**Comment by kamikazebr** [comment]
If it already deployed, its can be closed?

**Comment by rperez89** [comment]
> If it already deployed, its can be closed, or yet need review? If need i can review

was just deployed on staging to test, but needs review yes @0xGabi

**Review by tmoutinho — APPROVED** [review]

File: src/components/Onboarding/Screens/HoneyswapLiquidity.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ import { getNetwork } from '@/networks'
 import { toDecimals } from '@utils/math-utils'
 import { BYOT_TYPE } from '../constants'
 import { bigNum } from '@/lib/bigNumber'
+import BigNumber from '@lib/bigNumber'
```
Comment (reznord) [inline src/components/Onboarding/Screens/HoneyswapLiquidity.js:26]: Is this import required? Seems like we are not using it anywhere below

**Review by reznord — COMMENTED** [review]
Left one comment

**Review by reznord — APPROVED** [review]
LGTM

**Review by tmoutinho — APPROVED** [review]

**Comment by reznord** [comment]
hey @rperez89 , I was able to fix the eslint issues by disabling the `prettier/prettier` plugin and fixing the remaining 2 eslint errors. Should I fix that changes to the branch?

**Pull request merged**